baseapp-auth/baseapp_auth/rest_framework/register/serializers.py

```diff
@@ -0,0 +1,39 @@
+from django.contrib.auth import get_user_model
+from django.utils.translation import gettext as _
+
+from baseapp_referrals.models import UserReferral
```

Comment:
Something for the future, but I think we should decouple this module from any logic on the referrals module.